Tests/scripts/pkg_dev_tasks_in_docker.py

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+import argparse
+import yaml
+import glob
+import subprocess
+import os
+import hashlib
+import sys
+import shutil
+
+SCRIPT_DIR = os.path.dirname(os.path.abspath(__file__))
+CONTENT_DIR = os.path.abspath(SCRIPT_DIR + '/../..')
+sys.path.append(CONTENT_DIR)
+from package_creator import get_code_file  # noqa: E402
+
+DEF_DOCKER = 'demisto/python:1.3-alpine'
```

Comment:
shouldn't this be 1.4 ?

Reply:
No. Server is still using 1.3 as default